### PR TITLE
Added cli instructions to README

### PR DESCRIPTION
**Pull request by ritmun**

### CODE REVIEW

**Comment by boranx** [comment]
/retest
Any idea why the build jobs are stuck? Would it be possible to squash them to 1 commit?
Nice enhancements to the readme :+1: 
/lgtm

File: README.md [hunk 1]
```diff
@@ -64,9 +64,18 @@ err = cli.ValidateEgress(context.TODO(), "vpcSubnetID", "cloudImageID")
...
 ### Validate egress using command line
 
+Build the cli executable 
 ```shell
-AWS_ACCESS_KEY_ID=<redacted> AWS_SECRET_ACCESS_KEY=<redacted> ./osd-network-verifier egress --subnet-id subnet-0ccetestsubnet1864 --image-id=ami-0df9a9ade3c65a1c7
+make build
+```
+Execute 
+```shell
+AWS_ACCESS_KEY_ID=<redacted> AWS_SECRET_ACCESS_KEY=<redacted> ./osd-network-verifier egress --subnet-id <subnet-id> --image-id=<image-id>
```
Comment (fahlmant) [inline README.md:73]: Instead of redacted, could place something like <key> and <secret> here?

Comment (ritmun) [reply README.md:73]: I'm not sure what you mean by "something like and here". It was in the existing code but I can replace it with a random string.

**Review by fahlmant — APPROVED** [review]
/lgtm

**Pull request merged**